docs/world api.md

```diff
@@ -259,215 +262,124 @@ class MyGameWorld(World):
 
 ### Events
 
-Events will mark some progress. You define an event location, an
-event item, strap some rules to the location (i.e. hold certain
-items) and manually place the event item at the event location.
+An Event is a special combination of a Location and an Item, with both having an `id` of `None`. These can be used to
+track certain logic interactions, with the Event Item being required for access in other locations or regions, but not
+being "real". They may also be used for making the spoiler log look nicer, i.e. by having a `"Victory"` Event Item, that
+is required to finish the game, it will be very clear where the player finishes, rather than only seeing their last
+relevant found Item. Events function just like any other Location, and can still have their own access rules, etc.
+By convention, the Event "pair" of Location and Item typically have the same name, though this is not a requirement.
+They do not need to exist in the `name_to_id` lookups, as they will be stripped at the end of generation.
```

Comment:
```suggestion
They must not need to exist in the `name_to_id` lookups, as they will be stripped at the end of generation.
```
As requested by treble

Reply:
> as they will be stripped at the end of generation.

This is unclear imo. I would rephrase as something like

> They must not exist in the `name_to_id` lookups, are only used during generation and will not exist in the generated output.